suite-native/device-authentication/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "@suite-native/device-authentication",
+    "version": "1.0.0",
+    "private": true,
+    "license": "See LICENSE.md in repo root",
+    "sideEffects": false,
+    "main": "src/index",
+    "scripts": {
+        "lint:js": "yarn g:eslint '**/*.{ts,tsx,js}'",
+        "type-check": "yarn g:tsc --build"
```

Comment:
please rebase and add `depcheck` script please 🙏. You can get inspired in any other package. It is already merged in develop.

Reply:
[d947394](https://github.com/trezor/trezor-suite/pull/12889/commits/d94739451111f6544b19592ee56289ee31411928)